src/mem_usage.txt

```diff
@@ -0,0 +1,38 @@
+Release/RF24Log.elf:     file format elf32-avr
+00000314 l     F .text	00000006 _ZN20RF24StreamLogHandler11setLogLevelEh
+0000058a l     F .text	00000050 _ZN10RF24Logger4infoIPK19__FlashStringHelperEEvS3_T_z.constprop.15
+00800445 l     O .bss	00000002 rf24Logger
+000005da l     F .text	00000050 _ZN10RF24Logger5errorIPKcEEvPK19__FlashStringHelperT_z.constprop.14
+0000062a l     F .text	00000050 _ZN10RF24Logger4warnIPKcEEvPK19__FlashStringHelperT_z.constprop.13
+0000067a l     F .text	00000050 _ZN10RF24Logger4infoIPKcEEvPK19__FlashStringHelperT_z.constprop.12
+000006ca l     F .text	00000050 _ZN10RF24Logger5debugIPKcEEvPK19__FlashStringHelperT_z.constprop.11
+0000071a l     F .text	00000050 _ZN10RF24Logger5traceIPKcEEvPK19__FlashStringHelperT_z.constprop.10
+0000076a l     F .text	00000050 _ZN10RF24Logger5errorIPK19__FlashStringHelperEEvS3_T_z.constprop.9
+000007ba l     F .text	00000050 _ZN10RF24Logger4warnIPK19__FlashStringHelperEEvS3_T_z.constprop.8
+0000080a l     F .text	00000050 _ZN10RF24Logger5debugIPK19__FlashStringHelperEEvS3_T_z.constprop.7
+0000085a l     F .text	00000050 _ZN10RF24Logger5traceIPK19__FlashStringHelperEEvS3_T_z.constprop.6
+000008aa l     F .text	00000050 _ZN10RF24Logger3logIPKcEEvhPK19__FlashStringHelperT_z.constprop.5
+000008fa l     F .text	00000050 _ZN10RF24Logger3logIPK19__FlashStringHelperEEvhS3_T_z.constprop.4
+00000a3c l     F .text	00000220 _ZN20RF24StreamLogHandler12appendFormatEcPPv
+00000c5c l     F .text	00000086 _ZN20RF24StreamLogHandler14appendLogLevelEh
+00800100 l     O .data	00000006 _ZL17rf24logLevelTrace
+00800118 l     O .data	00000006 _ZL17rf24logLevelError
+00800112 l     O .data	00000006 _ZL16rf24logLevelWarn
+0080010c l     O .data	00000006 _ZL16rf24logLevelInfo
+00800106 l     O .data	00000006 _ZL17rf24logLevelDebug
+00000ce2 l     F .text	00000090 _ZN20RF24StreamLogHandler15appendTimestampEv
+00000d72 l     F .text	000000be _ZN20RF24StreamLogHandler3logEhPK19__FlashStringHelperS2_PPv
+00000e30 l     F .text	000000b0 _ZN20RF24StreamLogHandler3logEhPK19__FlashStringHelperPKcPPv
+00001024 l     F .text	0000007e _GLOBAL__I_65535_0_RF24DualLogHandler.cpp.o.1891
+00800440 l     O .bss	00000005 rf24SerialLogHandler
+00800130 l     O .data	0000000a _ZTV20RF24StreamLogHandler
+
+Core:
+.bss  0x002
+.data 0x000
+.text 0x3C0
```

Comment:
This is 960B. It may be drastically lowered down when removing those error(), warn(), info(), debug() and trace().

This is so called the core of the library and can not by optimized by the end user.

I would recommend to remove trace() as it is not needed when using sublevels of DEBUG.

Reply:
I vote yes to removing `trace()` as I've already expressed earlier.

Reply:
Trace() is removed.